resources-domain/resources-service/migrations/20230809143655-create-ResourcesAudits.js

```diff
@@ -0,0 +1,203 @@
+/**
```

Comment:
Can we follow the kebab case naming convention we use for the other migration scripts? thx